pytest-selenium/utils/utility.py

```diff
@@ -3,14 +3,27 @@
 
 class Library(object):
     def __init__(self):
-        self._book_slug_list = ["chemistry-2e", "chemistry-atoms-first-2e"]
+        self._book_dict = {
```

Comment:
it would be more clear to format this like

```
self._books = {
  "chemistry-2e": {
    "default_page":  "1-introduction"
  },
```

this is more extensible for when we want more stuff about the books, and includes a description about what the slug value is. the way it is now it would have to be named something like `_book_default_pages` but then also using it as the authority on what books are available is weird.

Reply:
Updated the book library